src/municipality/checkMuniticipalityOutput.ts

```diff
@@ -0,0 +1,34 @@
+import chalk from "chalk";
+import * as fs from "fs-extra";
+import * as path from "path";
+import { Municipality } from "../../definitions/Municipality";
+import { MUNICIPALITIES_OUTPUT_FOLDER } from "../config";
+
+// walk recursively inside all paths in dir and return the list of files met
+const walkSync = (dir: string): ReadonlyArray<string> => {
+  const files = fs.readdirSync(dir);
+  return files.reduce((aggr: ReadonlyArray<string>, curr: string) => {
+    const currentPath = path.join(dir, curr);
+    if (fs.statSync(currentPath).isDirectory()) {
+      return [...aggr, ...walkSync(currentPath)];
+    }
+    return [...aggr, currentPath];
+  }, []);
+};
+
+/**
+ * check if all json produced in the output folder are right encoded
+ */
+export const checkMunicipalityOutput = () => {
+  const jsons = walkSync(MUNICIPALITIES_OUTPUT_FOLDER).filter(j => {
+    try {
+      const obj = JSON.parse(fs.readFileSync(j).toString());
+      return Municipality.decode(obj).isLeft();
+    } catch {
+      return false;
```

Comment:
This should be `true`. For example if the json is malformed and the `JSON.parse` fails, the file should be invalid.

Reply:
@fabriziofff ops!

Reply:
fixed within https://github.com/pagopa/io-services-metadata/pull/121/commits/e8bde147610e766900e07c519b7f7dc8b378b255